src/views/Upload.vue

```diff
@@ -68,8 +78,33 @@ export default {
     uploadAllSamples() {
       this.$refs.uploadCards.forEach((card) => card.submitSample());
     },
+    dumpAllSamples() {
+      const samples = this.$refs.uploadCards.map((card) => card.dumpSample());
+      
+      if(samples.size == 0){
+        return;
+      }
+
+      const fields = Object.keys(samples[0]);
+      const replacer = function(nothing, value) { return value === null ? '' : value; };
```

Comment:
What do you think about turning this into an arrow function?

Reply:
Also, better use a "dangling underscore" (_) to represent unused parameters

Reply:
Done